C/impl/judy-1.0.5/sonLibJudySLArray.h

```diff
@@ -0,0 +1,52 @@
+#ifndef SONLIB_JUDYSLARRAY_H_
+#define SONLIB_JUDYSLARRAY_H_
+
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+#include "Judy.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif	
+
+
+int judy_equalKey(const void *key1, const void *key2);
+
+
+//Constructs an empty JudySL array
+Pvoid_t judy_construct();
```

Comment:
Unfortunately it isn't, an incompatible type 'void' error is thrown when I replace Pvoid_t with void